packages/api/src/authentication/authentication.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from "@nestjs/common";
+import { UsersService } from "../users/users.service";
+import { User } from "@cryptify/common/src/entities/user";
+import * as bcrypt from "bcrypt";
+import { JwtService } from "@nestjs/jwt";
+
+@Injectable()
+export class AuthenticationService {
+    constructor(private jwtService: JwtService, private usersService: UsersService) {}
+
+    async create(user: User): Promise<any> {
```

Comment:
I would call this reqUser and add a type to the return

```suggestion
    async create(reqUser: User): Promise<any> {
```

Reply:
already added Token type from previous comment, and your naming makes more sense so ill update it my code.